### DISCUSSION TITLE
Proxy Local部署模式下关于消息顺序性的疑问点

### DISCUSSION THREAD

**Discussion by wanghaiyang1221** [apache/rocketmq#8768] [Q&A]
在5.3.0版本下，采用Proxy Local部署方式时，关于顺序消息一致性问题：

1. 使用Java Client发送消息时，会从Proxy服务器获取注册中心上所有Topic主题配置，然后根据MessageGroup关键词，采用取模运算得到本次要调用的Proxy节点
2. 向Proxy节点发送消息数据，Proxy节点在队列选择逻辑中，通过getCurrentMessageQueueView方法构建的是本机上的队列，而没有使用到orderTopicConf这一选项配置

疑问：正如orderTopicConf配置的初衷所示，它是为了防止Broker节点掉线导致消息分发到不同的队列，保证了严格顺序而不是可用性，但在Proxy Local部署方式下，没有利用这一配置，那么是否会出现Broker节点掉线而导致消息分发到不同的队列这一情况呢？还是我有哪些点没有注意到？

**Comment by jaylu163** [comment]
老兄，你这边5.3.0版本的porxy使用local模式的时候，客户端发送消息可以成功吗。我这边遇到发送消息  ： task was cancelled
